exporter/stskafkaexporter/exporter_integration_test.go

```diff
@@ -0,0 +1,162 @@
+//go:build integration
+
+package stskafkaexporter
```

Comment:
Can you use the `_test` suffix for tests? The linter is going to scream even more later otherwise 😅 

```suggestion
package stskafkaexporter_test
```

Reply:
I just noticed the linter doesn't scream about this, and I don't precisely know why: if it's a big deal feel free to drop it :+1:

Reply:
Are there other options here? I noticed this linting rule, and it makes sense... but also, OTOH, I prefer to have functions like test helper to not be exposed, but with this linting rule, we're forced to make such functions public. 

I suppose making it public and assuming it won't be abused is a fine compromise 😀

Reply:
> I suppose making it public and assuming it won't be abused is a fine compromise
It is :smile: usually test helpers should go in the test, you can find all the rationales [on the official linter's page](https://github.com/maratori/testpackage). What I usually do in this case is declaring helpers in the test and letting the rule guide me through the refactoring of the package making sure that I'm not (for example) leveraging internals in tests :+1:

Reply:
Since the linter is not screaming at this, feel free to drop it.